public/styles/index.css

```diff
@@ -105,14 +108,23 @@
     }
     .__footerp {
         font-size: 16px;
-		}
-		
-		.__shecodelink {
-				padding: 10px 10px;
-		}
-		.__shecodenavbar {
-				padding: 0 20px;
-		}
+    }
+    .__shecodelink {
+        padding: 10px 10px;
+    }
+    .__shecodenavbar {
+        padding: 0 20px;
+    }
+    .__shecodeheader_title h1 {
+        font-size: 50px!important;
+    }
+    .__shecodeheader_subtitle h4 {
+        font-size: 20px!important;
+        margin-top: 20px!important;
+    }
```

Comment:
- Why are there a lot of `!important`s for header styles?
- Can we have a uniform font size for the different header types?

Reply:
If you study it closely, this is for enforcing responsive header styles on different screen sizes.